src/plugins/search/search.js

```diff
@@ -280,7 +279,7 @@ export function init(config, vm) {
 
   INDEXS = JSON.parse(localStorage.getItem(indexKey));
 
-  if (isExpired) {
+  if (isExpired || !INDEXS) {
```

Comment:
When manually delete `docsify.search.index` and keep `docsify.search.expires`, this plugin can still operate normally